Makefile

```diff
@@ -1,20 +1,26 @@
-TARGETS = naive cuda cupla kokkos oneapi
+TARGETS = naive cuda cupla kokkos oneapi alpakaCT alpakaRT
 
 .PHONY: all debug clean $(TARGETS)
 
+BOOST_BASE := /data/cmssw/slc7_amd64_gcc820/external/boost/1.67.0-pafccj/
+TBB_BASE := /data/cmssw/slc7_amd64_gcc820/external/tbb/2019_U3-pafccj
+
+TBB_FLAGS = -I$(TBB_BASE)/include -L$(TBB_BASE)/lib
 CXX := g++
-CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512
+CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512 -I$(BOOST_BASE)/include
```

Comment:
Why adding `-I$(BOOST_BASE)/include` in `ALPAKA_FLAGS` is not sufficient?

(I'd like to avoid the other cases to become dependent on boost)